dallinger/experiments/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""A home for all Dallinger Experiments. Experiments should be registered
+with a ``setuptools`` ``entry_point`` for the ``dallinger.experiments`` group."""
```

Comment:
Indeed, though I think a doc string in the module explaining what it's there for is appropriate and important. Where would the documentation for this go? http://dallinger.readthedocs.io/en/latest/required_experimental_files.html ?